test/Feature/OptionsSpec.hs

```diff
@@ -12,44 +12,66 @@ import SpecHelper
 
 spec :: SpecWith ((), Application)
 spec = describe "Allow header" $ do
-  it "includes read/write verbs for writeable table" $ do
-    r <- request methodOptions "/items" [] ""
-    liftIO $
-      simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET,POST,PUT,PATCH,DELETE,HEAD,OPTIONS"
-
-  it "includes read verbs for read-only table" $ do
-    r <- request methodOptions "/has_count_column" [] ""
-    liftIO $
-      simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET,HEAD,OPTIONS"
-
-  it "includes read verbs for non auto updatable views" $ do
-    r <- request methodOptions "/projects_view_without_triggers" [] ""
-    liftIO $
-      simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET,HEAD,OPTIONS"
-
-  it "includes read/write verbs for insertable, updatable and deletable views" $ do
-    r <- request methodOptions "/projects_view_with_all_triggers" [] ""
-    liftIO $
-      simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET,POST,PUT,PATCH,DELETE,HEAD,OPTIONS"
-
-  it "includes read and insertable verbs for insertable views" $ do
-    r <- request methodOptions "/projects_view_with_insert_trigger" [] ""
-    liftIO $
-      simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET,POST,HEAD,OPTIONS"
-
-  it "includes read and updatable verbs for updatable views" $ do
-    r <- request methodOptions "/projects_view_with_update_trigger" [] ""
-    liftIO $
-      simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET,PATCH,HEAD,OPTIONS"
-
-  it "includes read and insertable verbs for deletable views" $ do
-    r <- request methodOptions "/projects_view_with_delete_trigger" [] ""
-    liftIO $
-      simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET,DELETE,HEAD,OPTIONS"
+  context "a table" $ do
+    it "includes read/write verbs for writeable table" $ do
+      r <- request methodOptions "/items" [] ""
+      liftIO $
+        simpleHeaders r `shouldSatisfy`
+          matchHeader "Allow" "OPTIONS,GET,HEAD,POST,PUT,PATCH,DELETE"
+
+    it "includes read verbs for read-only table" $ do
+      r <- request methodOptions "/has_count_column" [] ""
```

Comment:
`has_count_column` is a view though, right?

Reply:
Yeah I think this test-case is redundant now. It's the same as: `it "includes read verbs for non auto updatable views" $ do`

Reply:
True, that shouldn't be there. I'll remove it entirely, that case is included in views tests.